### PR TITLE
Add EJS Supprt

### LINKED ISSUE #32: Require EJS templating Support
add EJS templating support in our app.

### PR DESCRIPTION
**Pull request by vivekshah1801**
Fixes #32

### CODE REVIEW

**Comment by YashKumarVerma** [comment]
Pull changes from master (check new travis file) to fix the build.

File: views/home.ejs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html> 
+<html> 
+<head> 
+    <title>Sample EJS</title>
+</head> 
+<body> 
+    <p>This is a EJS template.</p>
+    <p>read more @ https://ejs.co/</p>
+</body> 
+</html>
```
Comment (YashKumarVerma) [inline views/home.ejs:10]: load this somewhere in some file to ensure that this works.

Comment (vivekshah1801) [reply views/home.ejs]: added partial file for demo

**Comment by YashKumarVerma** [comment]
Please work on a clean branch from next time.

**Comment by YashKumarVerma** [comment]
Thanks a lot for your contributions.

**Pull request merged**